### ISSUE TITLE
Confusing and invalid JSON payload examples

### ISSUE THREAD

**Issue by m-mohr** [Open-EO/openeo-test-suite#47]
The test suite developed in WP4 contains invalid payloads:

For example: https://raw.githubusercontent.com/eodcgmbh/openeo-test-suite/11b37267ce409f05e6f8c50ee210602735580126/src/openeo_test_suite/tests/general/payload_examples/patch_jobs/patch_jobs_1.json (similarly post_result_1).

For example:
- `"subtype": "string"` => doesn't exist
- `"pattern": "/regex/"` => only string containing "regexp" allowed
- `"minimum": 0, "maximum": 0` both 0 => only 0 allowed
- the conditions allow conflict
- `"optional": false,` and `"default": null` at the same time don't work
- ...

Additionally, many things are very weirdly filled:
- summary: string?
- description: string?
- ...

The example(s) should be cleaned up.

**Comment by GeraldIr** [comment]
The payloads were taken directly from the openapi.json document (https://api.openeo.org/#tag/Data-Processing/operation/update-job), where I assumed that the given request samples were valid and I didn't look into it further.

I will update it with with a minimum working example that fits the constraints of the definition.

**Comment by m-mohr** [comment]
Yeah, unfortunately some of these examples are automatically generated by redoc and don't make a lot of sense. We should work on it and provide specific examples so that the autogeneration doesn't apply.
